balancer-js/src/modules/pools/pools.integration.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { expect } from 'chai';
+import { BalancerSDK, Network, PoolWithMethods } from '@/.';
+import { AddressZero, Zero } from '@ethersproject/constants';
+import { bn } from '@/lib/utils';
+
+const rpcUrl = 'http://127.0.0.1:8545';
+const network = Network.MAINNET;
+const sdk = new BalancerSDK({ network, rpcUrl });
+const { pools, contracts } = sdk;
+const { balancerHelpers } = contracts;
+
+const ethStEth =
+  '0x32296969ef14eb0c6d29669c550d4a0449130230000200000000000000000080';
+
+describe('pools module', () => {
+  describe('methods', () => {
+    let pool: PoolWithMethods | undefined;
+    before(async () => {
+      pool = await pools.find(ethStEth);
+    });
+
+    it('gets query function params', async () => {
+      if (!pool) {
+        return false;
+      }
+      const params = pool.buildQueryJoinExactIn({
+        sender: AddressZero,
+        amountsIn: [bn(1), Zero],
```

Comment:
Sorry to be repetitive about variable naming. I feel like these short and non-descriptive variables do not help reading the code. I have to keep reminding what they mean and even sometimes jump back to the definition to understand what they are doing.

Reply:
It's actually the same discussion as John raised below about `bn`. I'll resolve this and keep the discussion there.